app/views/branches/_form_expressions.html.erb

```diff
@@ -1,13 +1,10 @@
 <%= conditional.fields_for :expressions do |expression| %>
-  <div class="expression">
-    <div class="statement govuk-form-group" data-index-expression="<%= expression.index %>">
+  <div class="condition">
+    <div class="question govuk-form-group" data-index-expression="<%= expression.index %>">
```

Comment:
similarly here we're going to change this to `data-expression-index`

Reply:
I've just updated this to match your intentions (and the conflict correction of the other one).